addons/l10n_it_edi/models/account_invoice.py

```diff
@@ -37,8 +37,11 @@ def _get_l10n_it_amount_split_payment(self):
         amount = 0.0
         if self.is_invoice(True):
```

Comment:
```suggestion
        if self.is_sale_document(True):
```
I guess we could further filter the moves to sale documents only now?

Reply:
Yes. There is no purchase-type split payment tax.

Reply:
Wait, there is not at the moment, but it should be configurable, if you buy from a Public Administration business.

Reply:
Ok. I've updated the fix. :)

Reply:
After checking with Paolo, as it is only for XML export, only `out_invoice` and `out_refund` are handled.